Files/Views/LayoutModes/ColumnViewBrowser.xaml

```diff
@@ -15,6 +17,11 @@
     xmlns:tui="using:Microsoft.Toolkit.Uwp.UI"
     x:Name="PageRoot"
     mc:Ignorable="d">
+    <i:Interaction.Behaviors>
```

Comment:
This causes clicking anywhere on the page to invoke ``ItemPointerPressedCommand``, which means it will be invoked on middle click scrolling and when right clicking the layout. Can this not be handled in an item click event handler instead?

Reply:
> there is no reason to revert this feature from other layout modes.

I thought that was to only open a new folder tab if layout was ColumnView. I already corrected to work with all others as well as ColumnView

> This causes clicking anywhere on the page to invoke ItemPointerPressedCommand, which means it will be invoked on middle click scrolling and when right clicking the layout

About that issue i've tested and when right clicking the layout ItemPointerPressedCommand is not invoked. Only when middle button is pressed.